eng/SharedFramework.External.props

```diff
@@ -90,6 +90,7 @@
   -->
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <_CompilationOnlyReference Include="System.Buffers" />
+    <_CompilationOnlyReference Include="System.ComponentModel.Annotations" />
```

Comment:
Why is this here when System.ComponentModel.DataAnnotations is not?

Reply:
`System.ComponentModel.Annotations` is the name of the package that contains the `System.ComponentModel.DataAnnotations` types.